integration-tests/chaos/ocr_chaos_test.go

```diff
@@ -76,7 +76,14 @@ func TestOCRChaos(t *testing.T) {
 	}{
 		// network-* and pods-* are split intentionally into 2 parallel groups
 		// we can't use chaos.NewNetworkPartition and chaos.NewFailPods in parallel
-		// because of jsii runtime bug, see Makefile
+		// because of jsii runtime bug, see Makefile and please use those targets to run tests
```

Comment:
nit - maybe specify link to the bug to see when it will be fixed

Reply:
There is no issue opened, but `jsii.Kernel` just panics if you try another type of manifest sequentially